src/cli.c

```diff
@@ -97,6 +109,7 @@ static void print_help(void) {
            "  -v, --version                 print tjs version\n"
            "  -h, --help                    list options\n"
            "  -e, --eval EXPR               evaluate EXPR\n"
+           "  -l, --load FILENAME           module to preload (option can be repeated)\n"
```

Comment:
can you check what other runtimes have as the flag? QuickJS just add -I --include

Reply:
`node` uses `-r, --require`: https://nodejs.org/api/cli.html#cli_r_require_module because it basically does `require()` call under the hood while in our case we are dealing with esm imports. OTOH I couldn't find any equivalent `deno` flag. Haven't used other runtimes so my insights are quite limited 🤷‍♂

Reply:
qjs and tjs are nothing alike at this point, so let's go with your proposal!